### ISSUE TITLE
Rebuild ResNet CNN

### ISSUE THREAD

**Issue by brunofigura** [brunofigura/mlClassification#1]
## Die ResNet Architektur nachempfinden.

[ResNet from Scratch in Pytorch](https://blog.paperspace.com/writing-resnet-from-scratch-in-pytorch/)
- sollte alles covern

> ResNet könnte zu deep sein für MNIST Datensatz

**Comment by brunofigura** [comment]
## Paper dass dropouts erklärt. 
[Dropout: A Simple Way to Prevent Neural Networks from
Overfitting](https://jmlr.org/papers/volume15/srivastava14a/srivastava14a.pdf)
